lambda_code/ssm/Parameter/index.py

```diff
@@ -79,7 +79,15 @@ def validate(self):
         self.return_value_hash = strtobool(self.resource_properties.get('ReturnValueHash', 'false'))
 
     def attributes(self):
-        attr = {}
+        ssm = self.get_boto3_client('ssm')
+        parameter = ssm.get_parameter(
+            Name=self.name,
+            WithDecryption=False,
+        )
+
+        attr = {
+            'Arn': parameter['Parameter']['ARN']
```

Comment:
I think you can construct the Arn without an API call:
f"arn:aws:ssm:{region}:{account}:parameter{name}"

You can get both region & account from the context object.